src/cc/lib/benchmark/partition_benchmark.cc

```diff
@@ -62,7 +62,7 @@ static void BM_NODE_FEATURES(benchmark::State &state, snark::PartitionStorageTyp
     for (auto _ : state)
     {
         const size_t batch_size = state.range(0);
-        g_client->GetNodeFeature(std::span(input_nodes).subspan(offset, batch_size), std::span(features),
+        g_client->GetNodeFeature(std::span(input_nodes).subspan(offset, batch_size), {}, std::span(features),
```

Comment:
Did you check benchmarks remain the same after this change? Should a benchmark be added for timestamp filter?